### PR TITLE
new snapshot versions of modules

### PR DESCRIPTION
**Pull request by MalgorzataDzienia**
New snapshot versions are created for new development. Lets review them and merge them to develop

### CODE REVIEW

File: mrchecker-framework-modules/mrchecker-database-module/pom.xml [hunk 1]
```diff
@@ -59,7 +59,7 @@
     </scm>
 
     <properties>
-        <version>0.0.1-SNAPSHOT</version>
+
         
         <project.build.sourceEncoding>UTF8</project.build.sourceEncoding>
 	    <mrchecker-core-module.version>4.12.1.2</mrchecker-core-module.version>
```
Comment (michalbabiarz) [inline mrchecker-framework-modules/mrchecker-database-module/pom.xml]: Still old version of core

File: mrchecker-framework-modules/mrchecker-example-module/pom.xml [hunk 2]
```diff
@@ -7,7 +7,7 @@
     <parent>
         <artifactId>mrchecker-test-framework</artifactId>
         <groupId>com.capgemini.mrchecker</groupId>
-        <version>2.4.1</version>
+        <version>2.4.2-SNAPSHOT</version>
     </parent>
     <artifactId>mrchecker-example-module</artifactId>
     <version>${version}</version>
```
Comment (michalbabiarz) [inline mrchecker-framework-modules/mrchecker-example-module/pom.xml:13]: Line 52 (I'm unable to place a comment at that line) contains old core version.

**Review by michalbabiarz — CHANGES_REQUESTED** [review]

File: mrchecker-framework-modules/mrchecker-example-module/pom.xml (hunk 2 above)
Comment (MalgorzataDzienia) [reply mrchecker-framework-modules/mrchecker-example-module/pom.xml:13]: done

File: mrchecker-framework-modules/mrchecker-database-module/pom.xml (hunk 1 above)
Comment (michalbabiarz) [reply mrchecker-framework-modules/mrchecker-database-module/pom.xml]: I meant we should have: <mrchecker-core-module.version>4.12.1.4-SNAPSHOT</mrchecker-core-module.version>

**Review by michalbabiarz — APPROVED** [review]

**Pull request merged**